ec2system/ec2machine.go

```diff
@@ -375,11 +375,11 @@ func (s *System) Start(ctx context.Context, count int) ([]*bigmachine.Machine, e
 		return nil, fmt.Errorf("failed to marshal cloud-config: %v", err)
 	}
 	err = describeImages.Do(s.AMI, func() error {
-		out, err := s.ec2.DescribeImagesWithContext(ctx, &ec2.DescribeImagesInput{
+		out, errDescribe := s.ec2.DescribeImagesWithContext(ctx, &ec2.DescribeImagesInput{
```

Comment:
I'm not sure that carefully choosing `errDescribe`, `errWait`, etc. in these cases improves readability. I think there are cases in bigmachine/bigslice code where it's important to save an earlier error to return, etc., and using these names suggests that might be going on, or the different `errDescribe`s are related, which I think they are not.

I think it'd actually help readability to just given these a basic name like `err2`, or explicitly declare `var err error // nolint:...`, since I think that'll help a reader of idiomatic Go realize this just avoids shadowing.

Reply:
Okay; I converted most things to `err2` and reused `err` where the reuse made sense (by not using short declaration).